<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmConsulta</class>
 <widget class="QDialog" name="frmConsulta">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>471</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>652</width>
    <height>471</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>652</width>
    <height>471</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Consulta de Dados</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#lblConsulta
{
	font: 28px &quot;URW Gothic L&quot; ;
	color: white;
	font: bold;
}

.QFrame
{
	background-color: rgba(0, 0, 0, 100);
}

#btnCancelar
{
	background-color: rgb(170, 0, 0);
}

#btnEditar
{
	background-color: rgb(0, 0, 127);
}

#btnExcluir
{
	background-color: rgb(143, 143, 0);
}

#btnNovo
{
	background-color: rgb(0, 85, 0);
}</string>
  </property>
  <widget class="QLabel" name="lblConsulta">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic L</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CONSULTA DE DADOS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExcluir">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>410</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic L</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Excluir</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEditar">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>410</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic L</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Editar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNovo">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>410</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic L</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Novo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic L</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="gpbPesquisa">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>631</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit, QComboBox
{
   font: 12pt &quot;URW Gothic L&quot;;
}
QLineEdit
{
	background-color : rgb(0,0,0,100);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QComboBox" name="cmbParametro">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>301</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>301</width>
      <height>32</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>Par√¢metro*</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="txtPalavraChave">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>301</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>301</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>301</width>
      <height>32</height>
     </size>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Palavra(s)-chave*</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="gpbSessao">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>631</width>
     <height>271</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>631</width>
     <height>271</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit, QComboBox
{
   font: 12pt &quot;URW Gothic L&quot;;
}
QLineEdit
{
	background-color : rgb(0,0,0,100);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTableWidget" name="tblTabelaDados">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>611</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 45 10pt &quot;URW Gothic L&quot;;
color : white;
background-color: rgb(0,0,0,100);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::DashLine</enum>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>150</number>
    </attribute>
   </widget>
  </widget>
  <zorder>btnCancelar</zorder>
  <zorder>lblConsulta</zorder>
  <zorder>btnExcluir</zorder>
  <zorder>btnEditar</zorder>
  <zorder>btnNovo</zorder>
  <zorder>gpbPesquisa</zorder>
  <zorder>gpbSessao</zorder>
 </widget>
 <tabstops>
  <tabstop>cmbParametro</tabstop>
  <tabstop>txtPalavraChave</tabstop>
  <tabstop>tblTabelaDados</tabstop>
  <tabstop>btnCancelar</tabstop>
  <tabstop>btnEditar</tabstop>
  <tabstop>btnExcluir</tabstop>
  <tabstop>btnNovo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
